CONTRIBUTORS.txt

```diff
@@ -60,27 +68,30 @@ needs to be listed here.
 - John Connolly <connoljo2@gmail.com>
 - John Holdsworth <github@johnholdsworth.com>
 - John Lin <johnlinvc@gmail.com>
+- Joshua Rutkowski <54906829+joshrutkowski@users.noreply.github.com>
 - Jovan Milenkovic <jovan.milenkovic@htecgroup.com>
 - Julian Lettner <julian.lettner@apple.com>
 - Kaitlin Mahar <kaitlinmahar@gmail.com>
 - Kamil Borzym <kamil.borzym@gmail.com>
 - Karim ElNaggar <karimfarid.naggar@gmail.com>
 - Kevin Clark <kevin.clark@apple.com>
 - Kevin Sweeney <kevin_sweeney@apple.com>
+- Kim de Vos <kimdevos12@hotmail.com>
 - Konrad `ktoso` Malawski <ktoso@apple.com>
 - Kyle Bashour <kylebshr@me.com>
 - Kyle Browning <kylebrowning@me.com>
 - Kyle Jessup <kyle@perfect.org>
 - Lev Walkin <lwalkin@apple.com>
 - Liam Flynn <l.flynn@live.co.uk>
 - Ludovic Dewailly <ldewailly@apple.com>
+- Luo Xiu <luoxiustm@gmail.com>
 - Marcus Liotta <marcus@liotta.se>
 - Mario Sangiorgio <msangiorgio@apple.com>
 - Markus Jais <markusjais@googlemail.com>
 - Markus Kieselmann <markus@kieselmann.eu>
 - Marli Oshlack <marli.oshlack@apple.com>
 - Matt Eaton <agnosticdev@gmail.com>
-- Max Desiatov <max.desiatov@gmail.com>
+- Max Desiatov <max@desiatov.com>
```

Comment:
@MaxDesiatov I'm assuming the `max@desiatov.com` is your preferred one?

Reply:
That's correct, thanks!